Hippo.Core/Domain/OrderMetaData.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace Hippo.Core.Domain
+{
+    public class OrderMetaData
+    {
+        [Key]
+        [JsonIgnore]
+        public string Id { get; set; } = Guid.NewGuid().ToString();
```

Comment:
why not just use the regular identity id?

Reply:
Was just copying what sloth did :) 
I'm fine with just an int